modules/xml/src/main/java/org/mule/module/xml/util/XMLUtils.java

```diff
@@ -333,6 +333,19 @@ else if (obj instanceof byte[])
             // TODO Handle encoding/charset?
             return factory.createXMLStreamReader(new ByteArrayInputStream((byte[]) obj));
         }
+        else if (obj instanceof OutputHandler)
```

Comment:
this solution looks tied to the implementation of the current XML transformers which use this util class. Can be make it so that this is resolved by the generic transformer resolution algorithm?